podenv/defaults.py

```diff
@@ -148,4 +148,15 @@ def off(cap: str) -> Cap:
                   "libvdpau-va-gl", "mesa-dri-drivers",
                   "libva-intel-driver", "libva-intel-hybrid-driver"],
         home="~/.config/podhome/firefox"),
+
+    sshServer=dict(
+        parent="fedora",
+        capabilities=dict(on("network") + on("root")),
+        packages=["openssh-server"],
+        ports=[
+            "127.0.0.1:2022:22"
```

Comment:
That can now be parametrized with https://github.com/podenv/podenv/commit/69fa965ccd33f4bfca1c653725e887e3e409dfcf
Here is a new defaults that use that: https://github.com/podenv/podenv/commit/69fa965ccd33f4bfca1c653725e887e3e409dfcf